vector/src/main/java/im/vector/app/features/home/room/detail/timeline/TimelineEventController.kt

```diff
@@ -334,6 +337,10 @@ class TimelineEventController @Inject constructor(
                 .setVisibilityStateChangedListener(Timeline.Direction.FORWARDS)
                 .addWhenLoading(Timeline.Direction.FORWARDS)
 
+        val typingUsers = partialState.roomSummary?.typingUsers.orEmpty()
+        val typingItem = TypingItem_().id("typing_view").avatarRenderer(avatarRenderer).users(typingUsers)
+        add(typingItem)
```

Comment:
Maybe do not add the typing item if there is a forward loading item. So add the item only if `showingForwardLoader` is `false`.